src/components/Grid/grid-example.module.scss

```diff
@@ -1,13 +1,61 @@
+@import '../../nora/components/Colors.css';
+@import '../../nora/components/Attributes.css';
+
 :root {
   /**
    * CSS Properites can take form of var(--possibly-set-var, --default-fallback-var)
    * allowing us to declaratively theme our grid!
    */
-  --ariagrid-grid-container-color: #272727;
-  --ariagrid-grid-border-bottom: #f5f5f5;
-  --ariagrid-header-border-bottom: #d8dcde;
   --ariagrid-row-hover-background: var(--BrandDuckegg);
-  --ariagrid-grid-header-font-color: var(--GraySecondary--opaque);
+
+  /* Main grid font color */
```

Comment:
My thought is we can literally copy these root properties over to Nora and then adds a media-query backed container to support 1440 / 1920 as the grid's container is width: 100% so it'll respect whatever parent element it's in.